homeassistant/components/hive/strings.json

```diff
@@ -100,5 +100,19 @@
         }
       }
     }
+  },
+  "entity": {
+    "sensor": {
+      "heating": {
+        "state": {
+          "schedule": "Schedule"
+        }
+      },
+      "hot_water": {
+        "state": {
+          "schedule": "Schedule"
+        }
```

Comment:
Please also add the on and off ones, they should be references

Reply:
completed